src/constants/providerNames.ts

```diff
@@ -0,0 +1,2 @@
+export const DB_CONNECTION = 'DATABASE_CONNECTION';
+export const USER_REPOSITORY = 'USER_REPOSITORY';
```

Comment:
Do we need store constants that relate only to a specific module together in the root of the project?

Reply:
DB_CONNECTION relates to all modules
I think we have to keep all constants in one place, root dir
And it will be easier to find in by groups
But can keep in in one file

Reply:
Moved just to /constants/index.ts